ringpop.go

```diff
@@ -89,8 +90,9 @@ type Ringpop struct {
 		sync.RWMutex
 	}
 
-	logger log.Logger
-	log    log.Logger
+	logFacility logger.LogFacility
```

Comment:
`.log` and `.logger` were there from the beginning, as an early attempt to implement module logging from what I understand. There is no difference between them, and from what I remember only one gets the most use.

`.logFacility` is new, and isn't used anywhere yet. For now, to keep the code running, `log == logger == logFacility.Ringpop()` where `logFacility.Ringpop()` is a named logger.